test/spec/template/touch.js

```diff
@@ -0,0 +1,153 @@
+            var node = new Node({ template: tmpl });
+
+            assert(node.tmpl.testClick.style.cursor     === 'pointer');
+            assert(node.tmpl.testDblclick.style.cursor  === 'pointer');
+            assert(node.tmpl.testMouseover.style.cursor === 'pointer');
+            assert(node.tmpl.testMousemove.style.cursor === 'pointer');
+            assert(node.tmpl.testMousedown.style.cursor === 'pointer');
+            assert(node.tmpl.testMouseup.style.cursor   === 'pointer');
+          }
+        },
+        {
+          name: 'does not add cursor pointer for event-change',
+          test: function(){
+            var tmpl = createTemplate('<div{testElement} event-change="eventAction"></div>', true);
+            var node = new Node({ template: tmpl });
+
+            assert(node.tmpl.testElement.style.cursor === '');
+          }
+        },
+        {
+          name: 'does not overwrite existing value',
+          test: function(){
+            var tmpl = createTemplate('<div{testElement} event-click="eventAction" style="cursor: text"></div>', true);
+            var node = new Node({ template: tmpl });
+
+            assert(node.tmpl.testElement.style.cursor === 'text');
+          }
+        },
+        {
+          name: 'does not overwrite existing value',
+          test: function(){
+            var tmpl = createTemplate('<div{testElement} style="cursor: text" event-click="eventAction"></div>', true);
+            var node = new Node({ template: tmpl });
+
+            assert(node.tmpl.testElement.style.cursor === 'text');
+          }
+        },
+        {
+          name: 'reactive style - keeps cursor',
+          test: function(){
+            var tmpl = createTemplate('<div{testElement} style="color: {textColor}" event-click="eventAction"></div>', true);
+            var node = new Node({
+              template: tmpl,
+              binding: {
+                textColor: 'data:'
+              }
+            });
+
+            assert(node.tmpl.testElement.style.cursor === 'pointer');
+
+            node.update({ textColor: 'red' });
+
+            assert(node.tmpl.testElement.style.cursor === 'pointer');
+          }
+        },
+        {
+          name: 'reactive cursor - keeps specified',
+          test: function(){
+            var tmpl = createTemplate('<div{testElement} style="cursor: {cursor}" event-click="eventAction"></div>', true);
+            var node = new Node({
+              template: tmpl,
+              binding: {
+                cursor: 'data:'
+              }
+            });
+
+            assert(node.tmpl.testElement.style.cursor === 'pointer');
+
+            node.update({ cursor: 'text' });
+
+            assert(node.tmpl.testElement.style.cursor === 'text');
+
+            node.update({ cursor: 'pointer' });
+
+            assert(node.tmpl.testElement.style.cursor === 'pointer');
+          }
+        },
+        {
+          name: 'reactive cursor - keeps specified',
```

Comment:
`style binding for cursor property should work as expected (changed attribute order)`